docs/3.tutorials/fts/6-marketplace.md

```diff
@@ -42,7 +42,7 @@ Let's start by building both the finished FT contract and the marketplace contra
 
 
 ```bash
-cd market-contract && cargo near build && cd ..
+cd market-contract && cargo near build non-reproducible-wasm && cd ..
```

Comment:
in such occurencies where full command is required as part of a script, `non-reproducible-wasm` variant token was added to prevent prompts to used, 
 
it other places having only `cargo near build` prefix is still correct, as 2 choices will be suggested in interactive menu